### DISCUSSION TITLE
does it supposed to be like that? Next.js is slow or only for me?

### DISCUSSION THREAD

**Discussion by Choco-milk-for-u** [vercel/next.js#36829] [Help] [answered]
i haven't used next.js for a while, but i remember it was faster. Does it supposed to be like that.
![opera_ANeQRuUJU7](https://user-images.githubusercontent.com/84465375/167789955-2a3de889-b3be-4a12-b181-fccf4a07bd37.gif)
![lJ90wXgXaL](https://user-images.githubusercontent.com/84465375/167789963-837f5424-b379-4674-930c-bf72298d5a87.gif)

**Comment by icyJoseph** [comment, marked as answer]
Dev mode is not a good metric.

Though, 1 second is kinda fast for 11 thousand modules. I think the problem is that somehow you are using the entire MUI upfront. Impressive that it can compile that in 1 second. 

This would surely end up being a heavy bundle in production too. It is usually with icon libraries, that importing a simple icon, brings about the entire library of elements.

For example, I am not claiming this is the case but, it is often better to avoid:

```js
import {UserIcon} from '@some-icon-lib';
```

and do this instead:

```js
import {UserIcon} from '@some-icon-lib/icons/people/UserIcon';
```

Same with libraries with a ton of helpers.

**Reply by Choco-milk-for-u** [reply]
oh thats help, thanks
